tests/examples/wibox/awidget/prompt/highlight.lua

```diff
@@ -0,0 +1,66 @@
+local parent    = ... --DOC_NO_USAGE --DOC_HIDE
+local wibox     = require( "wibox"     ) --DOC_HIDE
+local awful     = { prompt = require("awful.prompt") }--DOC_HIDE
+local beautiful = require( "beautiful" ) --DOC_HIDE
+
+    local amp = "&amp"..string.char(0x3B)
+    local quot = "&quot"..string.char(0x3B)
```

Comment:
Yes, markdown requires 4 spaces for code. Some examples get away without it if they use `@usage`, but `@usage` is used differently for headers (as their documentation explains). This is why that example has a "DISABLE_USAGE" near the top. All lines with DOC_HIDE are hidden, so they don't need the indentation.

Also, before you ask, the ugly `string.char(0xFF)` is a bad hack around a bug. When I will rewrite this system to make @psychon happy, it wont use CMake, I can guarantee you that.